tzrec/constant.py

```diff
@@ -23,6 +23,8 @@ class Mode(Enum):
 
 EASYREC_VERSION = "0.7.5"
 
+EVAL_RESULT_FILENAME = "train_eval_result.txt"
```

Comment:
TRAIN_EVAL_RESULT_FILENAME = "train_eval_result_v2.txt"
prevent not compatible with old train_eval_result_v2.txt style